geocat/src/main/java/org/fao/geonet/geocat/services/reusable/Reject.java

```diff
@@ -83,6 +91,9 @@ public Element exec(Element params, ServiceContext context) throws Exception {
         boolean testing = Boolean.parseBoolean(Util.getParam(params, "testing", "false"));
         boolean isValidObject = Boolean.parseBoolean(Util.getParam(params, "isValidObject", "false"));
 
+        this.dataManager = context.getBean(DataManager.class);
```

Comment:
`@Autowired` ?

[edit] maybe not, actually, I cannot remember if a jeeves service is also a bean managed by spring.